src/raster/r.suitability.regions/r.suitability.regions.html

```diff
@@ -43,7 +43,7 @@ <h3>Option 2 - focal area</h3>
 neighborhood below. Using the maximum as statistic, the central cell 
 would be assigned a value of 5. Using the median, it would be assigned 
 the value 2. See <a 
-href="https://grass.osgeo.org/grass78/manuals/r.neighbors.html" 
+href="https://grass.osgeo.org/grass-stable/manuals/r.neighbors.html" 
```

Comment:
If we use grass-stable in the full links to the GRASS core manual pages, why are we setting the URLs to grass8 in the addons manual pages? (And in this PR, some are set to relative URLs)

Reply:
The best way to avoid hardcoded versions is using the "grass-stable" redirect on the server: modules with generic "grass-stable" URL currently still point to G78 but will be auto-fwd'ed on server to "grass80" as soon as G8.0.0 has been released.